spacing/_radius.scss

```diff
@@ -0,0 +1,4 @@
+:root {
+  --radius-none: 0;
+  --radius-full: 9999px;
```

Comment:
Lets leave it as is, although I think we would need to change this to some % value. It depends where it will be used.